src/ClientCapabilities.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace LanguageServer\Protocol;
```

Comment:
Since the `LanguageServer` namespace is defined in a different package I think this shouldn't "augment" it, instead it should define a top-level `LanguageServerProtocol` namespace

Reply:
Doing so would mean BC breaking changes for anybody who has extended PHP Language Server though (not sure if this is applicable).

Reply:
Updated though, it does make sense.

Reply:
That's okay, the versioning policy in the readme clearly states that semantic versioning is applied to the RPC interface, everything else are internals.